lab_5_scrapper/scrapper.py

```diff
@@ -100,6 +179,11 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    sleep(random.randrange(1, 5))
```

Comment:
is this range really needed for your webpage? try other ranges like 1-2, 1-3

some CI checks are red because they are unable to complete in a time given (see crawler checks output -> canceled operation)

Reply:
did it :)